integration-libs/epd-visualization/assets/translations/en/index.ts

```diff
@@ -0,0 +1,9 @@
+import { visualPicking } from './visual-picking';
+import { visualViewer } from './viewer';
+import { errors } from './errors';
+
+export const en = {
+  visualPicking,
+  visualViewer,
+  errors,
+};
```

Comment:
Please use one common i18n namespace for all the EPD feature. Ideally the namespace should indicate your feature, e.g.  

```
export const en = {
  epd,
};
```

or


```
export const en = {
  epdVisualisation,
};
```

Note: this will require also changing the usages of the keys:
e.g. `errors.visualLoad.noMatchingVisualFound` -> `epd.errors.visualLoad.noMatchingVisualFound`


The reason: in the current state, your feature "reserves" 3 global namespaces for itself. And those namespaces have quite common meaning (especially "errors"). Moreover, with enabled lazy-loading of JSON translation files, each namespace is a different JSON chunk. And we'd like to group all translations related to a particular feature in the user flow to one file. So grouping all your translations in `epd` key will result in mapping to one `/en/epd.json` lazy loaded translation file.

Reply:
Sure. I will update this.